packages/psss/src/views/AlertsOutbreaksSummaryView.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { WarningCloud, TabsToolbar, Virus } from '@tupaia/ui-components';
+import { Archive } from '@material-ui/icons';
+import {
+  Header,
+  HeaderTitleWithSubHeading,
+  OutbreaksExportModal,
+  AlertsExportModal,
+} from '../components';
+import { AlertsRoutes } from '../routes/AlertsRoutes';
+import { countryFlagImage } from '../utils';
+
+const links = [
+  {
+    label: 'Alerts',
+    to: '',
+    icon: <WarningCloud />,
+  },
+  {
+    label: 'Outbreaks',
+    to: '/outbreaks',
+    icon: <Virus />,
+  },
+  {
+    label: 'Archive',
+    to: '/archive',
+    icon: <Archive />,
+  },
+];
+
+export const AlertsOutbreaksSummaryView = () => {
+  const location = useLocation();
+  const ExportModal = location.pathname.includes('outbreak')
+    ? OutbreaksExportModal
+    : AlertsExportModal;
+
+  const Title = (
+    <HeaderTitleWithSubHeading
+      title="Alerts & Outbreaks"
+      subHeading="American Samoa"
+      avatarUrl={countryFlagImage('as')}
```

Comment:
We're probably at the point where you could update these based on the `match`

Reply:
I've updated the flag avatar to pull from the url. I haven't bothered with the country heading as I'd prefer to wait for the real country data for that one.